addon/lib/system/inflector.js

```diff
@@ -1,4 +1,5 @@
 import { capitalize } from '@ember/string';
+import { defaultRules } from "./lib/system";
```

Comment:
since we are already in `./lib/system` subdir we need to tweak this to `./inflections` (and remove `defaultRules` from `addon/lib/system.js` exports/imports...

Reply:
FWIW, this is why CI is failing